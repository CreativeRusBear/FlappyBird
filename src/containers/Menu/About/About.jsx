import React from 'react';
import classes from './About.module.css';
const About = (props) => {
    return (
        <div className={classes.About}>
            <h1>Author</h1>
            <p>This game was created by <a
                href='https://github.com/CreativeRusBear'
                target='_blank'
                rel='noopener noreferrer author'
            >
                CreativeRusBear (Artem Gusev)
            </a>
            </p>
        </div>
    );
}

export default About;