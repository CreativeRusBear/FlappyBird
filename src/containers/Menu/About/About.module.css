.About{
    display: flex;
    flex-flow: column;
    justify-content: center;
    align-items: center;
    color: var(--dark-brown-color);
}

.About a{
    color: var(--yellow-color);
    font-weight: 700;
}