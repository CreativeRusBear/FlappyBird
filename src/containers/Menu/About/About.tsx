import React from 'react';
import classes from './About.module.css';

const About = () => {
    return (
        <div className={classes.About}>
            <h1>Author</h1>
            <p>This game was created by <a
                href='https://github.com/corocoto'
                target='_blank'
                rel='noopener noreferrer author'
            >
                corocoto (Artem Gusev)
            </a>
            </p>
        </div>
    );
}

export default About;
