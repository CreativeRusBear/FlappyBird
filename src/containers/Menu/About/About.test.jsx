import React from "react";
import About from "./About";
import renderer from "react-test-renderer";

describe('<About />', () => {
  it('should renders correctly', () => {
    const tree = renderer
      .create(<About />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
