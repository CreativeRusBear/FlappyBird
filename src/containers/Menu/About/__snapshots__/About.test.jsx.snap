// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should renders correctly 1`] = `
<div
  className="About"
>
  <h1>
    Author
  </h1>
  <p>
    This game was created by 
    <a
      href="https://github.com/corocoto"
      rel="noopener noreferrer author"
      target="_blank"
    >
      corocoto (Artem Gusev)
    </a>
  </p>
</div>
`;
